theme/static/css/hydroshare_core.css

```diff
@@ -1473,12 +1473,16 @@ body > .container{
 
 #citation-text {
     display: inline-block;
-    float: left;
-	background:#EEE;
+    width: 100%;
+    min-height: 30px;
+    background: #EEE;
+    border: 1px solid #cccccc;
+    border-radius: 0px;
+    padding: 4px;
 }
 
 #citation-text a, .rights-url a{
-    word-break: break-word;
+    word-break:normal;
```

Comment:
The reason I was using 'word-break: break-word' is to keep the page responsive for mobile devices. Without some part of the input will be hidden when shrinking the page to the minimum.